nest/command_line.py

```diff
@@ -142,12 +145,12 @@ def _identity(x):
     # This is the command(s) passed to the command line utility
     cmd = args.command
 
-    if not os.path.exists(args.conf):
-        if args.client_id is None or args.client_secret is None:
-            print("Missing client and secret. Either call with --client-id "
-                  "and --client-secret or add to config as client_id and "
-                  "client_secret")
-            return
+    if args.client_id is None or args.client_secret is None:
```

Comment:
This works as expected-the issue in my earlier testing was that the documentation said `[DEFAULT]` could be used-but it can't.  So this has been modified back to the original check.